docs/kyma/docs/036-inst-custom-component-installation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Custom component installation
+type: Installation
+---
+
+Since Kyma is a modular tool, you can remove some components so that they do not install together with Kyma. You can also add some of them after the installation process. Read this document to learn how.
```

Comment:
Both versions are correct. If you prefer the second one, I do not mind changing it but personally, I prefer the first option since it is in active voice which is more acceptable from our guidelines perspective. We can discuss this linguistic issue later 😉

Reply:
ah ye indeed, I forgot about this eternal dis of passive voice :) let's keep it as it is